examples/esm-http-ts/README.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+This is a simple example that demonstrates tracing HTTP request, with an app written in TypeScript and transpiled to ES Modules.
```

Comment:
Is there a reason for having this sample on root and not at experimental/examples?

it seems also that it is missing in lerna.json

Reply:
Speaking of which, if we're good with this location, I should add it to the examples readme table

Reply:
> I just put the example where the other http and grpc examples were located. If we move examples around they should probably move together, so maybe a follow-up PR to limit scope here.

Sounds good. I'm good with keeping the example at this location, then moving the other examples also to the top-level in a follow-up PR. All of of the examples in the top-level are using experimental packages in some way, shape, or form anyway. :slightly_smiling_face: